bbb/services.py

```diff
@@ -519,6 +519,10 @@ def handlePending(self, data, msg):
         # as running. The BuildbotListener will take care of that when a slave
         # actually picks up the job.
         else:
+            redelivered = msg.get("delivery_info", {}).get('redelivered')
+            if redelivered:
+                log.warning("Task %s was redelivered. delivery_info: %s",
+                            taskid, msg.get("delivery_info"))
```

Comment:
Yes, this is to catch possible AMQP-related issues. TC-double-messages work is in progress in a different branch.